### ISSUE TITLE
'proto-gen' doesn't work

### ISSUE THREAD

**Issue by olimdzhon** [cosmosregistry/example#103]
Running MODULE_NAME=alice/checkers ./scripts/rename.sh on the repo root gives

~/minimal-module-example# MODULE_NAME=alice/checkers ./scripts/rename.sh
Generating protobuf files...
make: run: Command not found
make: *** [Makefile:31: proto-gen] Error 127

**Comment by nakul91** [comment]
@olimdzhon Please make sure docker is running as the rename.sh script uses docker.
this need to be added some where as its not there in any docs I reffered.

**Comment by olimdzhon** [comment]
@nakul91 are you sure? i can't see any docker command inside this script...

**Comment by nakul91** [comment]
@olimdzhon in Makefile they are using it

**Comment by olimdzhon** [comment]
@nakul91 yep you was right... thanks a lot

**Comment by olimdzhon** [comment]
started docker and it's worked

**Comment by olimdzhon** [comment]
@nikul91 
<img width="923" alt="Screenshot 2024-07-25 at 02 10 28" src="https://github.com/user-attachments/assets/abcdcd1e-6bf7-4f5a-b998-9f0c0a0d100c">
any idea why proto-gen not working? i have installed all prerequisites

**Comment by nakul91** [comment]
@olimdzhon add the authentic GitHub user name ex:  github.com/olimdzhon/checkers

**Comment by olimdzhon** [comment]
@nakul91 
<img width="951" alt="Screenshot 2024-07-25 at 11 11 11" src="https://github.com/user-attachments/assets/a26f6c1b-9d13-4aea-996e-71c412d60b02">
not working also...

**Comment by nakul91** [comment]
hard reset once and remove all untracked/stagged to commit  files once try -> git clean -fd and try rename

**Comment by olimdzhon** [comment]
@nakul91 
same results after reset and clean 
<img width="1072" alt="Screenshot 2024-07-25 at 11 33 23" src="https://github.com/user-attachments/assets/4489f681-63af-4601-8e3e-12ef7c375669">

**Comment by olimdzhon** [comment]
@nakul91 Sorry bro that i'm pinnig you each time))) i find out that protogen script ends with wrong commands and comment them...

<img width="499" alt="Screenshot 2024-07-25 at 12 19 05" src="https://github.com/user-attachments/assets/a98b384a-d9e3-4513-9d20-ca7f70dcbe51">

After that it tries connect to my github however i don't have such repo or modules... maybe if i setup ssh key and everything will work but i'm not sure...

<img width="1369" alt="Screenshot 2024-07-25 at 12 16 46" src="https://github.com/user-attachments/assets/f6206017-c010-4b35-9e5d-b0a7732d4e5e">

**Comment by olimdzhon** [comment]
@nakul91 If you mean docker itself... yes it's running...  

<img width="950" alt="Screenshot 2024-07-25 at 13 15 30" src="https://github.com/user-attachments/assets/28b4ae1f-5d3b-4b70-85a3-313ba9f3eb86">

i believe this docker run command from make file not executing because of errors...

<img width="706" alt="Screenshot 2024-07-25 at 13 14 16" src="https://github.com/user-attachments/assets/ae137072-c032-4801-be77-a73e537a18d4">

**Comment by nakul91** [comment]
before running
$make proto-gen

you have to comment out below line 
-   cp -r github.com/alice/checkers/* ./
+   # cp -r github.com/alice/checkers/* ./

check it here - https://tutorials.cosmos.network/hands-on-exercise/0-native/2-build-module.html

**Comment by olimdzhon** [comment]
yes i'm trying to learn from this tutorials, i tried both commented and commented out this lines in proto-gen but still can't generate modules because have this errors

**Comment by abuzaforfagun** [comment]
```
docker --version
Docker version 27.2.0, build 3ab4256
git status
On branch main
nothing to commit, working tree clean
```

When trying to run `make proto-gen` it throws following error:
```
Generating protobuf files...
process_begin: CreateProcess(NULL, /c/Program Files/Docker/Docker/resources/bin/docker run --rm -v C:/golang/minimal-module-example:/workspace --workdir /workspace ghcr.io/cosmos/proto-builder:0.14.0 sh ./scripts/protocgen.sh, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [proto-gen] Error 2
```
CLI: `Git bash`
OS: `Windows 11`

**Comment by julienrbrt** [comment]
It should be fixed https://github.com/cosmonity/example/commit/ac21eb6e9e77af68e4032a1f75af989b82fd9c18.
Let's not use the proto-builder anymore and install directly the local image
